dev/rstuf-bootstrap-payload.json

```diff
@@ -0,0 +1,90 @@
+{
+  "settings": {
+    "expiration": {
+      "root": 365,
+      "targets": 365,
+      "snapshot": 1,
+      "timestamp": 1,
+      "bins": 1
+    },
+    "services": {
+      "number_of_delegated_bins": 256,
```

Comment:
Should we choose a smaller number of delegations for easy debugging?

Reply:
The number should fit the expected number of target files. How many target files do we expect in a dev setup? If only a few for illustration, we can definitely start with a smaller number.

(ideally, the number is chosen automatically anyway, see e.g. https://gitlab.com/rugged/rugged/-/issues/137)

Reply:
Yes, it could be a feature in RSTUF, and it is ready to implement this feature with the last changes in the database model.
The current number in [PEP 458 is 16384](https://peps.python.org/pep-0458/#metadata-scalability). We can choose a smaller number, such as 16 or so.